docs/buildDocs.js

```diff
@@ -1,10 +1,17 @@
 const fs = require('fs').promises;
 const marked = require('marked');
-const mkdirp = require('mkdirp');
+const { mkdirp } = require('mkdirp');
 const path = require('path');
 const Prism = require('prismjs');
 const loadLanguages = require('prismjs/components/');
 
+marked.use(
+  // Turn off deprecated options. See https://github.com/markedjs/marked/issues/2793.
+  // For header IDs, these are implemented in our custom renderer below.
```

Comment:
Optional, maybe we want to use https://www.npmjs.com/package/marked-gfm-heading-id to get back the previous `headerIds` behaviour?

Reply:
Yeah, I tried that plugin but I liked the visible link on hover behavior of our custom renderer so figured I'd just adopt that behavior for all headers not just H3s.

Holler if that's a problem.